src/lib/snyk-to-html.ts

```diff
@@ -203,6 +203,9 @@ async function generateTemplate(data: any,
   data.uniqueCount = vulnMetadata.vulnerabilitiesUniqueCount;
   data.summary = vulnMetadata.vulnerabilitiesPathsCount + ' vulnerable dependency paths';
   data.showSummaryOnly = summary;
+  if(data.paths && data.paths.length == 1){
+    data.packageManager = data.paths[0].packageManager;
+  }
```

Comment:
Would be better to write as `data.paths?.length === 1`

Reply:
Yes, it will be good!